src/pages/ConfirmationCodePassword.js

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from "react"
+import { Row, Col, Container, ListGroup, Button, NavLink } from "react-bootstrap"
+import '../styles/pages.css'
+import apiUtil from '../utils/apiUtil'
+import { getUserState } from '../redux/selectors'
+import { useSelector, useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router-dom'
+import Notice from '../components/Notice'
+import { TailSpin } from  'react-loader-spinner'
+
+//we will need state variable to grab the email address.
+//onClick is going to redirect 
+
+//this function is prompting the user for email to send verification code
+function ConfirmationCodePasswordRequest() {
+    const [email, setEmail] = useState("")
+    const [ error, setError ] = useState(false)
+    const [ message, setMessage ] = useState("")
+    const dispatch = useDispatch()
+    const navigate = useNavigate()
+
+
+    
+    async function confirmationCodeRequest(emailPayload){
+        let response = {}
+
+        response = await apiUtil('put', '/users/password', {}, emailPayload)
+        console.log(response);
+
+        if(response.status == 200){
+            navigate("/reset/password")
+        }
+
+        setError(response.error)
+        setMessage(response.message)
+        setEmail("")
+    }
+
+
+    function emailObjectStaging(){
+
+        const emailInput = {
+            email: email
+        }
+
+        //console.log({emailInput})
+        confirmationCodeRequest(emailInput)
+
+    }
+
+    
+    return (
+    <div className="passwordDivContainer">
+        <h1 className="passwordPageh1">Confirmation Code</h1>
```

Comment:
I would title this something different. I was a little confused when I first got to this page because I thought it was asking for the code and not the email